client/src/components/Connections/components/Search.js

```diff
@@ -24,12 +24,13 @@ type Props = {
   ContextRouter
 
 const _searchConnection = (props: Props) => {
-  if (props.searchValue1.trim() === '' || props.searchValue2.trim() === '') {
-    return
+  if (props.searchValue1.trim() !== '' && props.searchValue2.trim().length === 0) {
+    props.history.push(`/prepojenia?eid1=${props.searchValue1.trim()}`)
+  } else if (props.searchValue1.trim() !== '' && props.searchValue2.trim().length > 0) {
+    props.history.push(
+      `/prepojenia?eid1=${props.searchValue1.trim()}&eid2=${props.searchValue2.trim()}`
```

Comment:
Just a nitpick but i don't like this if.
Value1 is checked twice and when length isn't === 0 its > 0, no need to check.